libchisel/src/remapimports.rs

```diff
@@ -271,4 +123,109 @@ mod tests {
         assert_eq!(output, expected);
         assert!(did_change);
     }
+
+    #[test]
+    fn remap_did_mutate() {
```

Comment:
Are all these tests passing with the old version of the module?

Reply:
these were not present in the old version of the module. Will have to check.

Reply:
Tested locally. They do work with the old code. Can we create a PR to merge tests in?

Reply:
Done in #80.